datahub-web-react/src/app/entity/shared/tabs/Documentation/components/DiscardDescriptionModal.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react';
+import { Modal, Button } from 'antd';
+import { useRouteToTab } from '../../../EntityContext';
+
+type Props = {
+    buttonProps?: Record<string, unknown>;
+    isDescriptionUpdated?: boolean;
+    urn: string;
+};
+
+export const DiscardDescriptionModal = ({ buttonProps, isDescriptionUpdated, urn }: Props) => {
+    const [isModalVisible, setIsModalVisible] = useState(false);
+    const routeToTab = useRouteToTab();
+    const localStorageDictionary = localStorage.getItem('editedDescriptions');
+    const editedDescriptions = (localStorageDictionary && JSON.parse(localStorageDictionary)) || {};
+
+    const showModal = () => {
+        if (isDescriptionUpdated) {
```

Comment:
Instead of doing this, we should use the "onComplete" function which is passed to the parent `DescriptionEditor` component. We should not route to tab inside of this modal component.